.github/workflows/release-scheduler.yml

```diff
@@ -1,32 +1,24 @@
 name: Release Scheduler
-
 on:
   schedule:
-    - cron: '15 15 * * MON' # Every Monday at 3:15pm UTC
+  - cron: '15 15 * * MON' # Every Monday at 3:15pm UTC
   workflow_dispatch:
-
-permissions:
-  contents: read
-
+permissions: read-all
 jobs:
   dispatch_scheduled_releases:
     name: Dispatch scheduled releases
-    if: ${{ github.repository == 'spring-projects/spring-security' }}
+    if: github.repository_owner == 'spring-projects'
```

Comment:
That should never happen. That would be an incredibly bad practice.

The reasoning for using the more general `github.repository_owner == 'spring-projects'` is so that it's easier to share/sync workflows across projects. Hard coding the repository name just adds one more unnecessary difference.